pkg/utils/uuid/uuid.go

```diff
@@ -76,6 +77,7 @@ func NewRandom() (UUID, error) {
 // data generated by h.  The hash should be at least 16 byte in length.  The
 // first 16 bytes of the hash are used to form the UUID.  The version of the
 // UUID will be the lower 4 bits of version.
+
```

Comment:
Do we need this line? Is it a lint suggestion?

Reply:
There shouldn't be a newline, I'll remove it.